modules/rosa-sdpolicy-networking.adoc

```diff
@@ -0,0 +1,65 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-service-definition.adoc
+
+[id="rosa-sdpolicy-networking_{context}"]
+= Networking
+
+== Custom domains for applications
+To use a custom hostname for a route, you must update your DNS provider by creating a canonical name (CNAME) record. Your CNAME record should map the OpenShift canonical router hostname to your custom domain. The OpenShift canonical router hostname is shown on the Route Details page after a Route is created. Or a wildcard CNAME record can be created once to route all subdomains for a given hostname to the cluster's router.
+
+== Domain validated certificates
+{product-title} includes TLS security certificates needed for both internal and external services on the cluster. For external routes, there are two, separate TLS wildcard certificates that are provided and installed on each cluster, one for the web console and route default hostnames and the second for the API endpoint. Let’s Encrypt is the certificate authority used for certificates. Routes within the cluster, e.g., the internal link:https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod[API endpoint], use TLS certificates signed by the cluster's built-in certificate authority and require the CA bundle available in every pod for trusting the TLS certificate.
+
+== Custom certificate authorities for builds
+{product-title} supports the use of custom certificate authorities to be trusted by builds when pulling images from an image registry.
+
+== Load Balancers
+{product-title} uses up to five different load balancers:
```

Comment:
For readability, I suggest adding an article to the beginning of each item. For example:
- An internal master load balancer
- An external master load balancer
...

Reply:
Done.